CMake Error:
ninja: error: build.ninja:1485: multiple rules generate ...
phonenumbers/metadata.h [-w dupbuild=err] 

Also Eliminate -Werror to fix sparc64 build

Index: cpp/CMakeLists.txt
--- cpp/CMakeLists.txt.orig
+++ cpp/CMakeLists.txt
@@ -351,7 +351,7 @@ add_metadata_gen_target (
   ${TEST_METADATA_TARGET}
   "${RESOURCES_DIR}/PhoneNumberMetadataForTesting.xml"
   "test_metadata"
-  "metadata"
+  "test_metadata"
 )
 list (APPEND TESTING_LIBRARY_SOURCES "src/phonenumbers/test_metadata.cc")
 
@@ -381,7 +381,7 @@ if (${USE_ICU_REGEXP} STREQUAL "ON")
 endif ()
 
 if (NOT WIN32)
-  add_definitions ("-Wall -Werror")
+  add_definitions ("-Wall")
 endif ()
 
 include_directories ("src")
